modules/gapi/include/opencv2/gapi/s11n.hpp

```diff
@@ -51,8 +51,60 @@ cv::GRunArgs deserialize(const std::vector<char> &p) {
     return detail::getRunArgs(p);
 }
 
-
 } // namespace gapi
 } // namespace cv
 
+namespace cv {
+namespace gimpl {
+namespace s11n {
```

Comment:
I'd make it `cv:gapi::s11n` or maybe `cv::gapi::S11N`. `gimpl` is our internal namespace standing for the implementation, and here we define an interface only

Reply:
Right, but the content of the `serialization.hpp` and `.cpp` files should be in the same namespace since we are using/overloading operators of the original interface. Should I make it `cv::gapi::s11n` in all files?

Reply:
Reworked namespaces